package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "map",
+  "description": "Kirby Map Field",
+  "author": "August Miller",
+  "version": "1.1.0",
+  "type": "kirby-plugin",
+  "license": "addMe"
```

Comment:
Mind updating this to include the recently added MIT license? Not sure what the format of the attribute should be.